Content.Server/GameObjects/Components/Kitchen/KitchenSpikeComponent.cs

```diff
@@ -27,50 +27,71 @@ void IActivate.Activate(ActivateEventArgs eventArgs)
         {
             SpriteComponent? sprite;
 
-            if (!EntitySystem.Get<SharedPullingSystem>().TryGetPulled(eventArgs.User, out var victim))
+            if (_meatParts == 0)
             {
-                if (_meatParts == 0)
-                {
-                    return;
-                }
-                _meatParts--;
+                return;
+            }
+            _meatParts--;
 
-                if (!string.IsNullOrEmpty(_meatPrototype))
+            if (!string.IsNullOrEmpty(_meatPrototype))
+            {
+                var meat = Owner.EntityManager.SpawnEntity(_meatPrototype, Owner.Transform.Coordinates);
+                if (meat != null)
                 {
-                    var meat = Owner.EntityManager.SpawnEntity(_meatPrototype, Owner.Transform.Coordinates);
-                    if (meat != null)
-                    {
-                        meat.Name = _meatName;
-                    }
+                    meat.Name = _meatName;
                 }
+            }
 
-                if (_meatParts != 0)
-                {
-                    eventArgs.User.PopupMessage(_meatSource1p);
-                }
-                else
+            if (_meatParts != 0)
+            {
+                eventArgs.User.PopupMessage(_meatSource1p);
+            }
+            else
+            {
+                if (Owner.TryGetComponent(out sprite))
                 {
-                    if (Owner.TryGetComponent(out sprite))
-                    {
-                        sprite.LayerSetState(0, "spike");
-                    }
-
-                    eventArgs.User.PopupMessage(_meatSource0);
+                    sprite.LayerSetState(0, "spike");
                 }
 
-                return;
+                eventArgs.User.PopupMessage(_meatSource0);
             }
 
+            return;
+            
+        }
+
+        public override bool DragDropOn(DragDropEventArgs eventArgs)
+        {
+            _ = TrySpike(eventArgs.Dragged, eventArgs.User);
```

Comment:
Do not discard task instances like this. Any exceptions will be rethrown in the finalizer thread which kills the server.

Make `TrySpike` `async void` instead, since you're not using the return value anyways.